packages/validator/test/__tests__/index.test.js

```diff
@@ -0,0 +1,280 @@
+import validator from '../../src';
```

Comment:
@vorobeez what are you think about splitting tests to separate files by field type?

Reply:
Will be implemented.